src/components/typings/Typeahead.ts

```diff
@@ -33,3 +33,18 @@ export interface TypeaheadState {
   value: string;
   showSuggestions: boolean;
 }
+
+type keys = Exclude<keyof TypeaheadProps, "onChange" | "children">;
+export type CachedTypeAheadProps = Pick<TypeaheadProps, keys> & {
```

Comment:
the whole code uses `interface` wherever possible.

Reply:
fixed by my commit

Reply:
Did not get it? Did you push?